### PR TITLE
fix: missing school year in scholarship certificate

### PR DESCRIPTION
**Pull request by haritianaadriano**

### CODE REVIEW

File: src/main/java/school/hei/haapi/service/utils/ScholarshipCertificateUtils.java [hunk 1]
```diff
@@ -31,18 +32,14 @@ private static String getAcademicYear(User student) {
...
   private static String specializationFiledString(User student) {
-    SpecializationField specializationField = student.getSpecializationField();
-    return switch (specializationField) {
-      case COMMON_CORE -> "Tronc commun";
-      case TN -> "Transformation Numérique";
-      case EL -> "Écosystéme Logiciel";
-      default -> throw new ApiException(SERVER_EXCEPTION, "Invalid specialization field");
-    };
+    String academicYear = getAcademicYear(student);
+    if (academicYear.equals("Pemière")) {
```
Comment (Mahefaa) [inline src/main/java/school/hei/haapi/service/utils/ScholarshipCertificateUtils.java]: typo

File: src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+import static java.time.Month.DECEMBER;
+import static java.time.Month.NOVEMBER;
+import static java.time.Month.OCTOBER;
+
+@Component
+public class SchoolYearUtils {
+  private static List<Month> startOfSchoolYear = List.of(
```
Comment (Mahefaa) [inline src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java]: list variable nouns are in plural

File: src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      OCTOBER,
+      NOVEMBER,
+      DECEMBER
```
Comment (Mahefaa) [inline src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java]: add January

Comment (Mahefaa) [reply src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java]: ask admins whether you should keep October or not

**Review by Mahefaa — CHANGES_REQUESTED** [review]

File: src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java (hunk 3 above)
Comment (haritianaadriano) [reply src/main/java/school/hei/haapi/service/utils/SchoolYearUtils.java]: are you sure to add January ? it's opposite of the logic of code no ?

**Pull request merged**